regions_www/tabs/manual_body.html

```diff
@@ -22,7 +22,7 @@ <h4><a name="manuals"></a>User Manuals</h4>
             <ul style="line-height:2.2;">
               <li>User Manual A: Overview (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-eng-user-manual-a.pdf">English</a>) (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-wel-user-manual-a.pdf">Welsh</a>)</li>
               <li>User Manual B: Using the PCT interface and data (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-eng-user-manual-b.pdf">English</a>) (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-wel-user-manual-b.pdf">Welsh</a>)</li>
-              <li>User Manual C1: PCT methods for the commuting layer (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-eng-user-manual-c1.pdf">English</a>) (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-wel-user-manual-c1.pdf">Welsh</a>)</li>
+              <li>User Manual C1: PCT methods for the commuting layer (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-eng-user-manual-c1.pdf">English</a>) (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-wel-user-manual-c1.pdf">Welsh</a>) (<a target="_blank" href = "https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-eng-user-manual-c1-yll-discounting.xlsx">Supplementary YLL discounting spreadsheet</a>)</li>
```

Comment:
just checked the discounting spreadsheet, looks fine:

![image](https://user-images.githubusercontent.com/1825120/69155266-4c269780-0ad9-11ea-9f5e-bab3d656da6a.png)

Reply:
Also just looked at the new manuals in English and Welsh, looking good.